java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -483,9 +485,15 @@ public CompletableFuture<Long> sortStore(
             @NonNull GlideString key,
             @NonNull GlideString destination,
             @NonNull SortOptionsBinary sortOptions) {
-        GlideString[] storeArguments = new GlideString[] {gs(STORE_COMMAND_STRING), destination};
+
         GlideString[] arguments =
-                concatenateArrays(new GlideString[] {key}, sortOptions.toGlideStringArgs(), storeArguments);
+                new ArgsBuilder()
```

Comment:
Please introduce a function which takes `GlideString...` argument, like we had before. It will simplify your code.